crates/matrix-sdk-common/src/deserialized_responses.rs

```diff
@@ -1247,4 +1248,126 @@ mod tests {
         let reason = UnableToDecryptReason::UnknownMegolmMessageIndex;
         assert!(reason.is_missing_room_key());
     }
+
+    #[test]
+    fn snapshot_test_verification_level() {
+        let level = VerificationLevel::VerificationViolation;
+        insta::assert_json_snapshot! {
+            serde_json::to_value(&level).unwrap(),
+        }
+
+        let level = VerificationLevel::UnsignedDevice;
+        insta::assert_json_snapshot! {
+            serde_json::to_value(&level).unwrap(),
```

Comment:
We have the choice here to inline the snapshot by using the `@""` token that insta.rs will replace with the actual string instead of generating a new file in `snapshots/`. WDYT?

Reply:
I find it nicer if it's like, with the snapshot out of sight.